iot_api/user_api/resources/Inventory.py

```diff
@@ -156,7 +156,7 @@ def get(self, asset_type, asset_id):
                 order_by=order_by,
                 page=page,
                 size=size,
-                alert_asset_type=AlertAssetType.DEVICE
+                asset_type=asset_type
             )
         else:
```

Comment:
I think this "if" block could be removed, right?

Reply:
Not really, the difference between these calls to find_with is that one of them sets the parameter device_id while the other one sets the parameter gateway_id